README.md

```diff
@@ -31,26 +32,51 @@ js-libp2p-floodsub
 > npm install libp2p-floodsub
 ```
 
-## Examples
+## Usage
 
 ```JavaScript
 const FloodSub = require('libp2p-floodsub')
 
-const fsub = new FloodSub(node)
+const registrar = {
```

Comment:
Should we just remove the registrar declaration here and make node that it is a libp2p Registrar? I know we don't have the docs flushed out for that yet, but we can point to that once it's done.

Reply:
I think you are right! Added a comment and removed the declaration. Once we have the docs, I will add a pointer to them here